www/front_src/src/Resources/Details/tabs/Notifications/Contacts.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+
+import { t } from 'i18next';
+import { isNil } from 'ramda';
+
+import { Box, Divider, IconButton, Paper, Tooltip } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+
+import {
+  labelConfigure,
+  labelNotEntitledAccessConfiguration,
+} from '../../../translatedLabels';
+
+import { ContactGroup, Contact } from './models';
+
+interface Props {
+  contacts: Array<Contact> | Array<ContactGroup>;
+  getColumns: (contact) => JSX.Element;
+  headers: JSX.Element;
+  templateColumns: string;
+}
+
+const Contacts = ({
+  contacts,
+  templateColumns,
+  getColumns,
+  headers,
+}: Props): JSX.Element => {
+  const goToUri = (uri): void => {
+    window.location.href = uri as string;
+  };
+
+  const getConfigurationColumn = ({ configuration_uri }): JSX.Element => {
+    const canGoToConfiguration = !isNil(configuration_uri);
+    const tooltipTitle = canGoToConfiguration
+      ? t(labelConfigure)
+      : t(labelNotEntitledAccessConfiguration);
+    const iconColor = canGoToConfiguration ? 'primary' : 'default';
+
+    return (
+      <Tooltip title={tooltipTitle}>
+        <IconButton
+          color={iconColor}
+          size="small"
+          sx={{ justifySelf: 'flex-end', marginRight: 1 }}
+          title={t(tooltipTitle)}
+          onClick={(): void => goToUri(configuration_uri)}
```

Comment:
Extract it to a specific function

Reply:
you talk about this function  goToUri  or onClick ? 

const goToUri = (uri): void => {
    window.location.href = uri as string;
  };